### ISSUE TITLE
Add NVM to list of possible solutions to permissions

### ISSUE THREAD

**Issue by ORESoftware** [npm/npm#15809]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [ ] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [x] Other (_see below for feature requests_):

#### What's going wrong?

Nothing right now :)

#### What's the feature?

I was thinking that NVM should be added to this list of solutions to use sudo to install packages globally:

https://docs.npmjs.com/getting-started/fixing-npm-permissions

Right now there is:

```
1. chown NPM root folders
2. change global installation dir to within user home dir
3. use homebrew
```

I am suggesting adding a 4th

`4. Use NVM`

However, perhaps using NVM is merely a case of # 2 in the list since I assume NVM uses this technique to do what it does? Still I think NVM should be addition, or mentioned in the second for changing the global installation dir.

I am an unbiased observer here, and do not have any affiliation with NVM other than I think it works pretty well.

**Comment by legodude17** [comment]
The thing is, if you want to mention `nvm`, you should also mention `n` and all the others that do the same thing.

**Comment by ORESoftware** [comment]
@legodude17, maybe although I think nvm is the best of the bunch by far. You can just add a 4th section that mentions these types of tools. Like I said, I think these also mostly belong in the category of "putting the npm global install location with user's $HOME". Just a suggestion, as I think NVM is pretty solid now.

TL;DR, using NVM is probably better than manually configuring the global npm install location, right?
